File: validate_email.js
const axios = require('axios');

/**
 * Validate an email address using ParheliaWeb API.
 * Get your free API key: https://parheliaweb.com/register
 */
const validateEmail = async (email, apiKey) => {
  try {
    const response = await axios.post(
      'https://parheliaweb.com/v1/email/validate',
      { email: email },
      {
        headers: {
          'x-api-key': apiKey,
          'Content-Type': 'application/json'
        }
      }
    );
    return response.data;
  } catch (error) {
    console.error('Validation error:', error.response?.data || error.message);
    throw error;
  }
};

// Example usage
const API_KEY = 'YOUR_API_KEY_HERE';
validateEmail('test@example.com', API_KEY)
  .then(result => console.log(result))
  .catch(err => console.error(err));

module.exports = { validateEmail };

File: validate_email.py
import requests

def validate_email(email, api_key):
    """
    Validate an email address using ParheliaWeb API.
    Get your free API key: https://parheliaweb.com/register
    """
    url = "https://parheliaweb.com/v1/email/validate"
    headers = {
        "x-api-key": api_key,
        "Content-Type": "application/json"
    }
    payload = {"email": email}
    
    response = requests.post(url, json=payload, headers=headers)
    return response.json()

# Example usage
if __name__ == "__main__":
    API_KEY = "YOUR_API_KEY_HERE"
    result = validate_email("test@example.com", API_KEY)
    print(result)
